events/importer/enkora.py

```diff
@@ -1784,16 +1809,28 @@ def keyword_ids_into_keywords(kw_ids: set) -> list[Keyword]:
 
         return kws
 
-    def convert_title(self, course: dict) -> tuple[str, str]:
+    @staticmethod
+    def convert_title(course: dict) -> tuple[dict, Optional[dict]]:
         """
         Add information to course title about event hour and weekday
         :param course: dict, Enkora course information as returned from API
-        :return: tuple[str, str], modified course title
+        :return: tuple[dict, Optional[dict]], dictionary of course titles
+        and modified course titles in supported languages
         """
-        course_title = course["reservation_event_group_name"].strip()
+        course_title = {}
+
+        for lang in EnkoraImporter.supported_languages:
+            course_title.update(
+                {lang: course.get(f"reservation_event_group_name_{lang}")}
+            )
+
+        # If there's no course title found for key: reservation_event_group_name_fi, set the default one
+        if not course_title["fi"]:
+            course_title.update({"fi": course.get("reservation_event_group_name")})
+
         if not course["reservation_events"]:
             # This should never happen. A course typically has events.
-            return course_title, course_title
+            return (course_title,)
```

Comment:
This will make [line 1610](https://github.com/City-of-Helsinki/linkedevents/pull/926/files#diff-a6ba5b39e3e6f6a6d823bbb4a758fded322fa143aa0fd14e5552c98f7a705192R1610) crash. So maybe 
```suggestion
            return (course_title, None)
```
?

Reply:
True, now corrected